docker-compose.yml

```diff
@@ -86,7 +86,7 @@ services:
       - "./docker/scratch/score-server-logs:/score-server/logs"
 
   score-client:
-    image: overture/score:5.0.0
+    image: overture/score:5.10.0
```

Comment:
Should this just be `latest` or `edge`?

Reply:
I have changed this to `latest` if we're looking for the latest stable version.